packages/checkbox/stories/Checkbox.stories.tsx

```diff
@@ -0,0 +1,330 @@
+    },
+    horizontal: {
+      checkboxLabel: {
+        display: 'inline-flex',
+        alignItems: 'center',
+        cursor: 'pointer',
+        color: 'gray',
+      },
+      checkboxWrapper: {
+        backgroundColor: baseBgColor,
+        paddingBottom: padding,
+        paddingLeft: padding,
+        paddingRight: padding,
+        paddingTop: padding,
+        margin: '10px',
+        borderRadius: padding,
+        color: textColor,
+        border: '1px solid green',
+      },
+      checkbox: {
+        backgroundColor: 'green',
+        borderRadius: '4px',
+        width: '14px',
+        height: '14px',
+      },
+      switchGroupWrapper: {
+        display: 'flex',
+        flexDirection: 'row',
+        flexWrap: 'nowrap',
+        paddingLeft: '5px',
+        paddingRight: '5px',
+        paddingTop: '10px',
+        paddingBottom: '10px',
+      },
+    },
+  },
+});
+
+const theme = generateTheme('2px', '#fff', '#000');
+
+const BasicCheckbox = ({
+  theme: innerTheme,
+  appearance,
+  type,
+  label,
+  checkIcon,
+  disabled = false,
+}: ICheckboxProps): React.ReactElement => {
+  const [isChecked, setIsChecked] = React.useState(false);
+
+  const handleChange = (): void => {
+    setIsChecked(!isChecked);
+  };
+
+  return (
+    <ThemeProvider theme={ innerTheme }>
+      <Checkbox
+        appearance={ appearance }
+        type={ type }
+        checked={ isChecked }
+        disabled={ disabled }
+        onChange={ handleChange }
+        label={ label }
+        checkIcon={ checkIcon }
+      />
+    </ThemeProvider>
+  );
+};
+
+const BasicSwitchGroup = ({
+  type,
+  theme: innerTheme,
+  appearance,
+  options,
+  disabled = false,
+  withAllGroup = false,
+}): React.ReactElement => {
+  const [values, setValues] = React.useState([options[0].value]);
+  const [isAll, setIsAll] = React.useState(false);
+
+  const handleChange = (value: (string | number)[]): void => {
+    setValues(value);
+    setIsAll(false);
+  };
+
+  const handleAllChange = React.useCallback((checked: boolean) => {
+    if (checked) {
+      setValues(options.map((option: IOption) => option.value));
+    } else {
+      setValues([]);
+    }
+
+    setIsAll(checked);
+  }, [options]);
+
+  return (
+    <ThemeProvider theme={ innerTheme }>
+      {
+        type === 'checkbox' && withAllGroup && (
+          <Checkbox
+            appearance={ appearance }
+            label="Choose all"
+            checked={ isAll }
+            disabled={ disabled }
+            onChange={ handleAllChange }
+          />
+        )
+      }
+      <SwitchGroup
+        type={ type }
+        appearance={ appearance }
+        onChange={ handleChange }
+        options={ options }
+        values={ values }
+        disabled={ disabled }
+      />
+    </ThemeProvider>
+  );
+};
+
+storiesOf('Checkbox', module)
+  .add('Basic', () => (
+    <div>
+      <GlobalStyle />
+      <BasicCheckbox
+        theme={ { [checkboxThemeNamespace]: theme } }
```

Comment:
It is enough to add custom theme in 1 example. It is unnecessary put theme in every story.